pkg/cmd/server/kubernetes/master.go

```diff
@@ -107,6 +115,24 @@ func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string
 	recycler.Run()
 }
 
+// attemptToLoadRecycler tries decoding a pod from a filepath for use as a recycler for a volume.
+// If a path is not set as a CLI flag, no load will be attempted and no error returned.
+// If a path is set and the pod was successfully loaded, the recycler pod will be set on the config and no error returned.
+// Any failed attempt to load the recycler pod will return an error.
+// TODO: make this func re-usable upstream and use downstream.  No need to duplicate this function.
+func attemptToLoadRecycler(path string, config *volume.VolumeConfig) error {
+	glog.V(5).Infof("Attempting to load recycler pod file from %s", path)
```

Comment:
it is unfortunate that none of our relative path resolution logic will be able to run on the filename provided in the args... it'll only be able to be an absolute path or a path relative to the binary working dir